Implement #149 - Verify route long name and route short name unicity (#351)
MobilityData/gtfs-validator: RULES.md

### W012 - Optional csv file is empty

Empty csv optional file found in the archive: file contains header but does not have data.  
This is related to [E047](https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md#E047).

<a name="W014"/>

### W014 - Duplicate `routes.route_long_name`

> All routes should have different `route_long_name`. If routes have same `route_long_name`, they must be different routes serving different areas; and must not be different trips of the same route or different directions of the same route.

Reviewer: ```suggestion
All routes should have different `route_long_name`. If routes have the same `route_long_name`, they must be different routes serving different areas; and must not be different trips of the same route or different directions of the same route.
```

<a name="W015"/>

### W015 - Duplicate `routes.route_short_name`

> All routes should have different `route_short_name`. If routes have same `route_long_name`, they must be different routes serving different areas; and must not be different trips of the same route or different directions of the same route.

Reviewer: ```suggestion
All routes should have different `route_short_name`. If routes have the same `route_short_name`, they must be different routes serving different areas; and must not be different trips of the same route or different directions of the same route.
```

<a name="W016"/>

> ### W016 - Duplicate combination of fields `route_long_name` and `routes.route_short_name`

Reviewer: ```suggestion
### W016 - Duplicate combination of fields `routes.route_long_name` and `routes.route_short_name`
```

> `route_short_name` and `route_long_name` 'should not be used for more than once. 

Reviewer: ```suggestion
The same combination of `route_short_name` and `route_long_name` should not be used for more than one route.
```
